ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -2574,8 +2871,8 @@ db $22, $9B, $E1, $00
 
 endif
 
-; Loads the animated tiles after the overworld map is closed.
 pullpc
+; Loads the animated tiles after the overworld map is closed.
```

Comment:
If the jump to DecompOwAnimatedTiles is no longer needed here, should we just remove it? What does this comment and removed function offer that we need to retain it? 

Additionally, I'm curious if it may be a good idea to preallocate a function for any custom graphics and leave that function call uncommented wherever it goes, so the size of the script is consistent?

E.g. 

```
; Some free space 
ZSOW_LoadCustomGraphics:
  ; Maybe register push/pops or leave that to users
  ; User defined custom graphics code
  RTL
```

Reply:
I think i thought that this was something Vanilla was doing but looking at the vanilla function now it seems that this was something I added. So in context leaving a comment saying that vanilla was wrong would be worthwhile but I'll remove it now.

as for the prellocated function is a good idea. I'll put a TODO for that